src/plugins/intel_gna/src/log/gna_plugin_log.hpp

```diff
@@ -16,8 +16,6 @@
 #define PLOT
 #define GNA_HEAP_PROFILER
 #define MODEL_DUMP
```

Comment:
I think this define is not used in code at all

Reply:
which one?